Packs/HelloWorld/Integrations/FeedHelloWorld/Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "==5.0.1"
+pytest-mock = "*"
+requests-mock = "*"
+pytest-asyncio = "*"
+
+[packages]
+pytest = "*"
+requests = "*"
+
+[requires]
+python_version = "3.7"
```

Comment:
```suggestion
python_version = "3.9"
```

Reply:
The script creates all the required files for when creating a feed integration, and a README.md file is one of those. The reason it is named "README_example.md" (and not "README.md") is because if it was as the first option, it would have been shown in our [docs]([url]( https://xsoar.pan.dev/docs/reference/index)) site (and we want to avoid that). The script handles that and changes the name after the creation, see in 'initiator.py' in the sdk repository (line 592).

Reply:
I still don't understand why we don't have a README for the integration like is done in the HelloWorld integration?

Reply:
I discussed that with Dan and he suggested ill use the same logic that is used for HelloIAMWorld, as it is a similar concept template (from the same reason, that we dont want to show the README file in the docs site). As this README file is empty anyway, II can change it to README.md and, since it is empty, it wont be shown in the docs site either way. What do you think?

Reply:
Let's keep it the same as the HelloIAMWorld integration